cloud-common/src/main/java/com/sequenceiq/cloudbreak/util/ClientUtil.java

```diff
@@ -75,6 +74,23 @@ public static Client createClient(String serverCert, String clientCert, String c
         return client;
     }
 
+    public static Client createClient() {
```

Comment:
In an other class restCLient.java we have the same thing and it does not require extra cert parameters, we need to merge the ClientUtil.java and RestClient.java to one class to avoid code duplication: https://github.com/sequenceiq/cloudbreak/blob/0b6f20f83e9bdb8ae2425cd119d0a97c6526a023/core-api/src/main/java/com/sequenceiq/cloudbreak/client/RestClient.java